lib/buy/dfx/dfx_buy_provider.dart

```diff
@@ -70,6 +70,24 @@ class DFXBuyProvider extends BuyProvider {
     }
   }
 
+  @override
+  List<WalletType> getSupportedWalletTypes(bool isBuy) {
+    if (isBuy) {
+      return [
+        WalletType.bitcoin,
+        WalletType.ethereum,
+        WalletType.polygon,
+        WalletType.monero,
+        WalletType.nano,
```

Comment:
DFX doesn't support nano, can you please make sure that you have migrated the correct wallet types to each provider